packages/shared/src/components/Feed.tsx

```diff
@@ -329,14 +334,20 @@ export default function Feed<T>({
         postMenuLocation.row,
         postMenuLocation.column,
       ),
-    onBookmark: () =>
+    onBookmark: () => {
+      const bookmarked = !!(items[postMenuIndex] as PostItem)?.post?.bookmarked;
+      const toastMessage = bookmarked
+        ? 'Post successfully removed from your bookmarks'
+        : 'Post successfully added to your bookmarks';
+      displayToast(toastMessage);
```

Comment:
This will now always show it, also if you are shown the login modal.
I think we should only fire the toast when you actually bookmark it.

Reply:
Fixed